sdk/go/common/util/logging/log.go

```diff
@@ -43,8 +43,28 @@ var LogFlow = false     // true to flow logging settings to child processes.
 var rwLock sync.RWMutex
 var filters []Filter
 
-func V(level glog.Level) glog.Verbose {
-	return glog.V(level)
+type VerboseShim glog.Verbose
```

Comment:
RE: name: How about one of "VerboseLogger", "VerboseLog", "VLog", or "Chatty"?

Naming-wise, `logging.*Log` does stutter a bit, but it also doesn't look like the type is intended to be used directly much so it's not a problem.

Reply:
Yeh I'll probably go with `VerboseLogger` is the most clear, and yes most uses of this means no one actually uses the type name.